### ISSUE TITLE
Set timezone inside VNM to the timezone of the host machine

### ISSUE THREAD

**Issue by civier** [NeuroDesk/neurodesk#55]
Right now it is set to UTC. I'm using VNM version 20210130

**Comment by stebo85** [comment]
yes, good idea and I noticed this as well, but I don't know how to find out in which timezone the host machine is. We would need a command line argument potentially when starting it up? Any ideas anyone?

**Comment by stebo85** [comment]
Idea how to fix this: 
- https://www.wikihow.com/Change-the-Timezone-in-Linux
- get timezone of user via command line argument of docker container
- capture in startup.sh and set there
- also choose cvmfs mirror based on timezone
